chains/orchestrator-relays/runtime/dancelight/src/bridge_to_ethereum_config.rs

```diff
@@ -362,6 +541,13 @@ impl snowbridge_pallet_inbound_queue::Config for Runtime {
     type MessageProcessor = (
         SymbioticMessageProcessor<Self>,
```

Comment:
Are we sure symbiotic message processor does not pick this command before the `EthTokensLolcaProcessor` does?